tests/cmake/CMakeLists.txt

```
@@ -12,12 +12,14 @@ add_subdirectory(
   "${CMAKE_CURRENT_SOURCE_DIR}/../../DevGuideExamples/DCPS/Messenger/build")
 
 if(NOT ${CMAKE_VERSION} VERSION_LESS "3.8.2")
-  add_subdirectory(Messenger)
-  add_subdirectory(Nested_IDL)
-  if(NOT OPENDDS_STATIC)
-    add_subdirectory(generated_global)
-  endif()
-  if(NOT OPENDDS_SAFETY_PROFILE AND OPENDDS_CXX11)
-    add_subdirectory(C++11_typecode)
+  if(OPENDDS_OWNERSHIP_PROFILE)
+    add_subdirectory(Messenger)
+    if(NOT OPENDDS_STATIC)
+      add_subdirectory(generated_global)
+    endif()
+    if(NOT OPENDDS_SAFETY_PROFILE AND OPENDDS_CXX11)
+      add_subdirectory(C++11_typecode)
+    endif()
   endif()
```

```suggestion
  endif()
  if(NOT OPENDDS_SAFETY_PROFILE AND OPENDDS_CXX11)
    add_subdirectory(C++11_typecode)
  endif()
```

As far as I can tell typecode doesn't need ownership.

---

I think my brain was crosswired and saw this as C++11_Messenger. I'll fix it.